app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateWalletFragment.kt

```diff
@@ -141,55 +145,15 @@ class CreateWalletActivity : BaseActivity() {
         createWalletButton.visibility = View.GONE
```

Comment:
Annotation for this function should be `@OnClick(R.id.create_wallet_btn)`. Now it causes a glitch when I click the button area multiple times.